docs/how_to/namespaces/quotas.md

```diff
@@ -0,0 +1,44 @@
+---
+version: 3.2.0
```

Comment:
It took us so long so 3.2.0 was actually released. Let's change it to 3.3.0 and we can merge it :-)

Reply:
Sure, sorry about that, work has been very busy

Reply:
Updated :)